aws_signing_helper/signer.go

```diff
@@ -101,10 +85,11 @@ type CredentialProcessOutput struct {
 	Expiration string `json:"Expiration"`
 }
 
-type RolesAnywhereSigner struct {
-	PrivateKey       crypto.PrivateKey
-	Certificate      x509.Certificate
-	CertificateChain []x509.Certificate
+type CertificateContainer struct {
+	// Certificate data
+	Cert *x509.Certificate
+	// Certificate URI (only populated in the case that the certificate is a PKCS#11 object)
+	Uri string
```

Comment:
Not relevant to the OS cert store, correct? What's the implication of leaving it in?